xtask/rules_check/src/lib.rs

```diff
@@ -356,28 +304,25 @@ fn assert_lint(
                         let error = diag
                             .with_file_path(test.file_path())
                             .with_file_source_code(code);
-                        let res = diagnostics.write_diagnostic(error);
-
-                        // Abort the analysis on error
-                        if let Err(err) = res {
-                            eprintln!("Error: {err}");
-                            return ControlFlow::Break(err);
-                        }
+                        diagnostics.write_diagnostic(error);
                     }
 
-                    ControlFlow::Continue(())
+                    ControlFlow::<()>::Continue(())
                 });
             }
         }
         DocumentFileSource::Css(..) => {
-            let parse = biome_css_parser::parse_css(code, CssParserOptions::default());
+            let parse_options = CssParserOptions::default()
+                .allow_css_modules()
+                .allow_tailwind_directives();
```

Comment:
The CSS parser options are updated here too: https://github.com/biomejs/biome/pull/7544

I'm thinking we may want to do even more code sharing here...